src/structures/idt.rs

```diff
@@ -340,8 +340,37 @@ pub struct InterruptDescriptorTable {
     /// vector nr. 20
     pub virtualization: Entry<HandlerFunc>,
 
-    /// vector nr. 21-28
-    reserved_2: [Entry<HandlerFunc>; 8],
+    /// A #CP exception is generated when shadow stacks are enabled and mismatch
+    /// scenarios are detected (possible error code cases below).
+    ///
+    /// The error code is the #CP error code, for each of the following situations:
+    /// - A RET (near) instruction encountered a return address mismatch.
+    /// - A RET (far) instruction encountered a return address mismatch.
+    /// - A RSTORSSP instruction encountered an invalid shadow stack restore token.
+    /// - A SETSSBY instruction encountered an invalid supervisor shadow stack token.
```

Comment:
This exception can also be generated due to a missing ENDBRANCH instruction if indirect branch tracking is enabled.